### PR TITLE
BN Rules Update: Adding minimum activity

### PR DESCRIPTION
**Pull request by NoffyNoffeh**
This follows the discussion thread held at https://osu.ppy.sh/community/forums/topics/1448027?n=1 . It adds a bottom line for BN activity, and clarifies the time range for how activity is checked. 

Please leave open for at least 48h while to review this one last time, to make sure it's accurate to intended changes. Thank you!

- [x] approval from other NAT members

### CODE REVIEW

File: wiki/People/The_Team/Beatmap_Nominators/Rules/en.md [hunk 1]
```diff
@@ -18,8 +18,11 @@ Sustained behaviour in conflict with these expectations is grounds for dismissal
 
 ## Activity
 
-- **Activity requirements for your respective gamemode(s) must be met.** Average activity over the last 3 months will be checked on the first day of each month and during regular BN evaluations, taking into account any leave of absence communicated. Any BN who falls below their mode's required average will get an activity warning.
-- **When warned for activity, minimum activity requirements for your respective gamemode(s) must be met over the course of one month.** Failing to meet the required minimum when being warned for it prior may result in removal from the Beatmap Nominators.
+- **Activity requirements for your respective gamemode(s) must be met.** Activity over the last 90 days will be checked on the first day of each month and during regular BN evaluations, taking into account any leave of absence communicated.
```
Comment (Walavouchey) [inline wiki/People/The_Team/Beatmap_Nominators/Rules/en.md:21]:
```suggestion
- **Activity requirements for your respective game mode(s) must be met.** Activity over the last 90 days will be checked on the first day of each month and during regular BN evaluations, taking into account any leave of absence communicated.
```

File: wiki/People/The_Team/Beatmap_Nominators/Rules/en.md [hunk 2]
```diff
@@ -18,8 +18,11 @@ Sustained behaviour in conflict with these expectations is grounds for dismissal
...
+  - **Falling below 60 days worth of nominations in a 90 day period will result in a removal when activity is checked**. For example, an osu! BN with less than 6 nominations in a 90 day period will be removed.
+  - **Falling below the 90 day requirements with more than 60 days worth of nominations will result in an activity warning.** 
+  - When warned for activity, minimum activity requirements for your respective gamemode(s) must be met over the course of one month.
```
Comment (Walavouchey) [inline wiki/People/The_Team/Beatmap_Nominators/Rules/en.md:24]:
```suggestion
  - When warned for activity, minimum activity requirements for your respective game mode(s) must be met over the course of one month. 
```

File: wiki/People/The_Team/Beatmap_Nominators/Rules/en.md [hunk 3]
```diff
@@ -18,8 +18,11 @@ Sustained behaviour in conflict with these expectations is grounds for dismissal
 
 ## Activity
 
-- **Activity requirements for your respective gamemode(s) must be met.** Average activity over the last 3 months will be checked on the first day of each month and during regular BN evaluations, taking into account any leave of absence communicated. Any BN who falls below their mode's required average will get an activity warning.
-- **When warned for activity, minimum activity requirements for your respective gamemode(s) must be met over the course of one month.** Failing to meet the required minimum when being warned for it prior may result in removal from the Beatmap Nominators.
+- **Activity requirements for your respective gamemode(s) must be met.** Activity over the last 90 days will be checked on the first day of each month and during regular BN evaluations, taking into account any leave of absence communicated.
+  - **Falling below 60 days worth of nominations in a 90 day period will result in a removal when activity is checked**. For example, an osu! BN with less than 6 nominations in a 90 day period will be removed.
```
Comment (Walavouchey) [inline wiki/People/The_Team/Beatmap_Nominators/Rules/en.md:22]:
```suggestion
  - **Falling below 60 days worth of nominations in a 90 day period will result in a removal when activity is checked.** For example, an osu! BN with less than 6 nominations in a 90 day period will be removed.
```

**Comment by Walavouchey** [comment]
What's the status of this? Have other NAT members looked at this yet?

**Comment by NoffyNoffeh** [comment]
Yep, should be good to go.

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**